### PR TITLE
Remove segmentCount from the default configurations

### PR DESCRIPTION
**Pull request by adejanovski**
Update the doc with segmentCountPerNode and add token range coalescing explanations.

### CODE REVIEW

File: docs/docs/configuration/docker_vars/index.html [hunk 1]
```diff
@@ -584,7 +590,7 @@ <h2 id="direct-mapping-to-reaper-specific-configuration-settings">Direct Mapping
 
 <tr>
 <td><code>REAPER_LOGGING_FORMAT</code></td>
-<td><code>REAPER_LOGGING_APPENDERS_CONSOLE_LOG_FORMAT</code></td>
+<td><code>REAPER_LOGGING_APPENDERS_LOG_FORMAT</code></td>
```
Comment (ossarga) [inline docs/docs/configuration/docker_vars/index.html]: This is being changed incorrectly. I may have missed this fix when I was updating the documentation.

It should be: `REAPER_LOGGING_APPENDERS_CONSOLE_LOG_FORMAT`.

Sorry about that. Should I fix this before we merge this PR?

File: src/docs/content/docs/concepts.md [hunk 2]
```diff
@@ -10,9 +10,12 @@ weight = 5
 
 Reaper splits repair runs in segments. A segment is a subrange of tokens that fits entirely in one of the cluster token ranges. The minimum number of segments for a repair run is the number of token ranges in the cluster. With a 3 nodes cluster using 256 vnodes per node, a repair run will have at least 768 segments. If necessary, each repair can define a higher number of segments than the number of token ranges.
 
+**Starting with Reaper 1.2.0 and Cassandra 2.2, token ranges that have the same replicas can be coalesced into a single segment.** If the total number of requested segments is lower than the number of vnodes, Reaper will try to group token ranges so that each segment has the appropriate number of tokens.
```
Comment (ossarga) [inline src/docs/content/docs/concepts.md]: Consider the following modification to the start of the sentence.

> As of Reaper 1.2.0 and Apache Cassandra 2.2, ...

**Review by ossarga — APPROVED** [review]

**Comment by adejanovski** [comment]
Hey Anthony feel free to make all changes that seem appropriate and merge the PR.

Thanks!

**Comment by ossarga** [comment]
No worries Alex. Changes made. Waiting for CI to go green before merging.

**Pull request merged**